shared/reducers/chat2.js

```diff
@@ -820,6 +832,7 @@ const rootReducer = (
     case Chat2Gen.messageDelete:
     case Chat2Gen.messageEdit:
     case Chat2Gen.messageWasEdited:
+    case Chat2Gen.pendingMessageWasEdited:
```

Comment:
??

Reply:
What's wrong here?

Reply:
Is that the same key?

Reply:
The first one is in `metaMapReducer` the second is in `rootReducer`